lib/licensee/projects/github_project.rb

```diff
@@ -19,10 +19,12 @@ class GitHubProject < Licensee::Projects::Project
 
       class RepoNotFound < StandardError; end
 
-      def initialize(github_url, **args)
+      def initialize(github_url, revision: nil, **args)
         @repo = github_url[GITHUB_REPO_PATTERN, 1]
         raise ArgumentError, "Not a github URL: #{github_url}" unless @repo
 
+        @revision = revision
```

Comment:
Similar to the above comment, wondering if it makes sense to add `revision` as a named arg on the parent class and raise in the file system project if it's passed?